doc/governance/index.rst

```diff
@@ -0,0 +1,72 @@
+Ceph Governance
+---------------
+
+Key principles
+==============
+
+ * Decision-making is consensus-driven by those who show up.
+ * Leadership roles are defined primarily by responsibility, not prestige or seniority.
+   * It is normal and healthy for these roles to be passed on to others
+ * Everyone's role is ultimately to serve the users and participation
+   is voluntary.
+
+Bodies
+------
+
+Ceph Executive Council
+======================
+
+Responsibilities
+^^^^^^^^^^^^^^^^
+
+ * Arbiter in cases where decisions cannot be reached by consensus
+ * Distribute key responsibilities amongst themselves or others
+ * Point of contact for the project
+ * Representatives for Ceph foundation board meetings
+ * Ensure things get done
+
+Membership
+^^^^^^^^^^
+
+ * 3 people
+ * Elected by steering committee
+   * Candidates self-nominate or are nominated by other members
+   * Discussion of how roles/responsibilities may be delegated
+   * Ranked-choice vote by steering committee
+ * 2 year terms, with one member staggered
```

Comment:
I'll rephrase this - I was trying to summarize @robbat2's simple offset suggestion:

Summary: Simple Offset:

first 2 positions: every 2 years
last position: first ever instance gets 1-year term, then it's every 2 years after that.
In case of early retirement/stepping-down/death etc, that position is available again (could lead to all 3 positions needed to be filled again)

Reply:
Understood now!